fs-ansible/roles/lustre/tasks/mount.yaml

```diff
@@ -44,12 +44,19 @@
       with_items: "{{ mdts.keys() }}"
 
     - name: mount MDTs
-      command: mount -t lustre /dev/{{ item }}p1 /lustre/{{ fs_name }}/MDT/{{ item }}
+      command: mount -t lustre /dev/mapper/dac--{{ item }}-mdt /lustre/{{ fs_name }}/MDT/{{ item }}
       register: command_result
       failed_when: "command_result.rc != 0 and ('is already mounted' not in command_result.stderr)"
       changed_when: "command_result.rc == 0"
       with_items: "{{ mdts.keys() }}"
 
+    - name: Enable DNE2 remote dir
```

Comment:
should we not only do this once, not for all servers?

Reply:
It does only need to be set on the MGS so it can be updated to depend only on the MGS variable being defined and not MDT